source/clear-linux/concepts/telemetry-about.rst

```diff
@@ -47,11 +47,13 @@ Architecture
 The telemetry client provides the front end of a complete telemetrics solution
 and includes the following components:
 
-*	**telemd**, a daemon that prepares the records to send to a telemetrics server or
-	spools the records on disk in case it cannot successfully deliver them.
-*	Probes that collect specific types of data from the operating system.
+*	**telemprobd**, a daemon that prepares the telemetry records and spools them on disk prior to delivery
+* **telempostd**, a daemon that takes the records, verifies and then sends to the telemetry server or spools on disk in case the server is unavailable.
```

Comment:
The records are already spooled aren't they?  telempostd just sends them along or deletes them if the get too old

Reply:
It depends on the configuration setting -- if configured to store locally instead of send, then telempostd stores them on disk as specified.

Reply:
telemprobd stores them in the spool directory and they stay in that location and then telempostd with try to send them to the server if record_server_delivery_enabled is set to true and the server is alive.  Telempostd will also delete a record after it expires (with the config option record_expiry value) or keep the record if record_retention_enabled is set to true.